.travis.yml

```diff
@@ -159,10 +161,10 @@ jobs:
             --volume $TRAVIS_BUILD_DIR:/root/travis
             --env OS=centos-stream8
             --env DIST=el8
-            --env LIBDAP_RPM_VERSION=$LIBDAP_RPM_VERSION
-            --env BES_BUILD_NUMBER=$BES_BUILD_NUMBER
-            --env AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID
-            --env AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
+            --env LIBDAP_RPM_VERSION
```

Comment:
Is this shorthand for 'Pass in the env var of the same name?'

And, this doesn't have anything to do with the change log hack, correct?